server-embedded/src/main/java/org/exoplatform/chat/services/mongodb/ChatMongoDataStorage.java

```diff
@@ -534,6 +537,47 @@ public void setRoomName(String room, String name) {
     }
   }
 
+  @Override
+  public boolean isRoomEnabled(String room) {
+    boolean isEnabled = true;
+    DBCollection coll = db().getCollection(M_ROOMS_COLLECTION);
+
+    BasicDBObject basicDBObject = new BasicDBObject();
+    basicDBObject.put("_id", room);
+
+    DBCursor cursor = coll.find(basicDBObject);
+    if (cursor.hasNext()) {
+      try {
+        DBObject doc = cursor.next();
+        if (doc.get("isEnabled") != null) {
+          isEnabled = (StringUtils.equals(doc.get("isEnabled").toString(), "true"));
+        } else {
+          doc.put("isEnabled", true);
+          coll.save(doc, WriteConcern.UNACKNOWLEDGED);
+        }
+      } catch (MongoException me) {
+        LOG.warning(me.getCode() + " : " + room + " : " + me.getMessage());
```

Comment:
This should be an error and not a warning

Reply:
Okay will fix it